src/components/Item.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+
+export default function Item({ deleteItem, item, editItem }) {
+  // create state thats going to control your conditionals
+  const [editing, setEditing] = useState(false);
+
+  // create varible that will change depending on the state
+  let itemContent;
+
+  if (editing) {
+    itemContent = (
+      <form
+        onSubmit={(e) => {
+          e.preventDefault();
+          setEditing(false);
+        }}
+      >
+        <input
+          value={item.text}
+          onChange={(e) =>
+            editItem({
+              ...item,
+              text: e.target.value,
+            })
+          }
+        />
+        <button type="submit">Save Changes</button>
+      </form>
+    );
+  } else {
+    itemContent = (
+      <>
+        <button aria-label={`edit ${item.text}`} onClick={() => setEditing(true)}>
+          Edit
+        </button>
+      </>
+    );
+  }
+
+  // write if statement that will render elements depending on state
+
+  return (
+    <div>
```

Comment:
If the div isn't being used for style and only as a way to return one element, use a fragment <></> instead